docs/crd_mysqlcluster_v1beta2.md

```diff
@@ -136,7 +136,7 @@ PersistentVolumeClaim is a user's request for and claim to a persistent volume.
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
 | metadata | Standard object's metadata. | [ObjectMeta](#objectmeta) | true |
-| spec | Spec defines the desired characteristics of a volume requested by a pod author. | [corev1.PersistentVolumeClaimSpec](https://pkg.go.dev/k8s.io/api/core/v1#PersistentVolumeClaimSpec) | true |
+| spec | Spec defines the desired characteristics of a volume requested by a pod author. | PersistentVolumeClaimSpecApplyConfiguration | true |
```

Comment:
Please update https://github.com/cybozu-go/moco/blob/main/docs/links.csv to link to the appropriate API manual.
In this case, it is https://pkg.go.dev/k8s.io/client-go@v0.23.3/applyconfigurations/core/v1#PersistentVolumeClaimApplyConfiguration

Reply:
Thanks you, I fixed:
https://github.com/cybozu-go/moco/pull/364/commits/41fa72ed465531c07620e198f7e6a49e994473c4